### PR TITLE
Add includes file for registry.json config

### PR DESCRIPTION
**Pull request by usha-mandya**
The registry.json file creation process is currently documented in three topics. Creating an includes file to single source this section.

### CODE REVIEW

**Review by StefanScherer — DISMISSED** [review]
LGTM

File: _includes/configure-registry-json.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<!-- This section is included in topics that contain instructions on how to configure registry.json file to enforce users to sign into Docker Desktop-->
+
+## Create a registry.json file
+
+After you’ve successfully installed Docker Desktop, create a `registry.json`
+file. Before you create a `registry.json` file, ensure that the user is a member
```
Comment (jerae-duffin) [inline _includes/configure-registry-json.md:6]: developer instead of user

File: _includes/configure-registry-json.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+On macOS, you must create a file at `/Library/Application Support/com.docker.docker/registry.json` with file permissions that ensure that
+the developer using Docker Desktop cannot remove or edit the file (that is, only
+the system administrator can write to the file). The file must be of type `JSON`
+and contain the name of the Docker Hub organization names in the `allowedOrgs`
+key.
+
+To create your `registry.json` file on macOS:
+
+1. Navigate to VS Code or any text editor of your choice.
+2. Enter the name of the Docker Hub organization that the user belongs to in the
```
Comment (jerae-duffin) [inline _includes/configure-registry-json.md:44]: developer

File: desktop/index.md [hunk 3]
```diff
@@ -55,16 +55,20 @@ For information about Docker Desktop licensing, see [Docker Desktop License Agre
...
 Authenticated users get a higher pull rate limit compared to anonymous users. For example, if you are authenticated, you get 200 pulls per 6 hour period, compared to 100 pulls per 6 hour period per IP address for anonymous users. For more information, see [Download rate limit](../docker-hub/download-rate-limit.md).
 
-Authenticating through Docker Desktop also allows administrators to configure Docker Desktop and set up guardrails using the [Image Access Management](../docker-hub/image-access-management.md) feature to allow team members to:
-
-* Only have access to Trusted Content on Docker Hub
-* Pull only from the specified categories of images
-
-To authenticate using Docker Desktop, click the **Sign in/Create ID** option on the Docker menu and follow the on-screen instructions.
+In large enterprises where admin access is restricted, administrators can create
+a `registry.json` file and deploy it to the users’ machines using a device
```
Comment (jerae-duffin) [inline desktop/index.md:65]: developers'

File: desktop/index.md [hunk 4]
```diff
@@ -55,16 +55,20 @@ For information about Docker Desktop licensing, see [Docker Desktop License Agre
...
+management software as part of the Docker Desktop installation process. Enforcing users to authenticate through Docker Desktop also allows
```
Comment (jerae-duffin) [inline desktop/index.md:66]: same here

**Review by jerae-duffin — APPROVED** [review]
LGTM

**Pull request merged**